packages/explorer/src/bin/explorer.ts

```diff
@@ -143,6 +171,9 @@ process.on("SIGINT", () => {
   if (explorerProcess) {
     explorerProcess.kill();
   }
+  if (indexerProcess) {
+    indexerProcess.kill();
+  }
   process.exit();
```

Comment:
it does not, does indexer spawn subprocesses?

Reply:
By subprocesses I meant the things you're `spawn`ing here. I expect since this explorer bin process is the one spawning, it has a parent-child relationship, and would kill any of its children if the parent process got killed. But maybe that assumption is why a lot of my tools end up with hanging/zombie processes 🙈 

Down to use something like terminate, but usage seems low, so I wonder if there's a more mainstream/battle-tested variant out there.

Reply:
Heh, yeah `terminate` package does not seem widely used, and might be an overkill. Just looked into it a bit more and based on my understanding, keeping track of child processes, and then killing them once the parent process is killed is the way to do it. A common danger is spawning child processes but then not killing them explicitly once parent process is killed but we handle that with `process.on("SIGINT")` listener so should be good there. Then the child processes, once stopped, should themselves stop their own child processes but looks like we have no problems there since both `explorer` and `store-indexer` don't seem to leave anything hanging.

Reply:
what about other signals like sigkill?

Reply:
Damn, thought I had everything. So turns out `SIGKILL` cannot be captured but `SIGTERM` can. So now capturing `SIGINT` and `SIGTERM` which captures it all to the best of my knowledge. Will do a bit more research into this

Reply:
doesn't help with sigkill but looks like we can listen to both sigint and sigterm with
```ts
process.on('exit', () => {
  ...
})
```

Reply:
Just tried it and it's working. Checked the docs and it says `The 'exit' event is emitted when the Node.js process is about to exit as a result of either:` so it does make sense to clean up the child processes here based on the documentation, updated.